adagrams/game.py

```diff
@@ -1,11 +1,81 @@
+import random 
+import copy
+
+letter_pool_frequency = {
```

Comment:
Because these variables don't get re-assigned, it might be good to make them constants